packages/web-config-server/src/apiV1/dataBuilders/generic/reportServer/reportServerDataBuilder.js

```diff
@@ -39,6 +39,10 @@ export class ReportServerBuilder extends DataBuilder {
 
     const response = await this.reportConnection.fetchReport(this.config.reportCode, requestQuery);
 
+    // Matrix table data
+    if (response.results.columns && response.results.rows) {
+      return { ...response.results };
+    }
```

Comment:
We should return {columns: {}, rows: {}, ...} instead of {data: {columns: {}, rows: {}}, ...} . For bar chart we need to return {data: {...}, ...}.

Ideally we can let `report server` output step to handle that. But we can do it in the future if we need a new functionality for bar chart in report-server's output step?

Reply:
Ahh good point... man these inconsistencies can be soo annoying hey 😝 

I would say that yeah, this should all actually just be handled by the report-server output step, but obviously that's too large a refactor right now so lets create a ticket for it?

Reply:
I think it is pretty straight forward. I will let report-server handle bar chart output in this PR, if you don't mind reviewing it?